README.md

```diff
@@ -374,6 +374,17 @@ bundle install
 bundle exec rake test
 ```
 
+or if you'd rather use docker just run:
+```bash
+docker run -it --name shopify_api -v $PWD:/shopify_api -w="/shopify_api" ruby:2.4.6 bundle install
```

Comment:
I think its wise to quote the volume arg so it's more portable to different shells: `"$PWD:/shopify_api"`